testing/architecture-test/src/changes/accepted-changes/accepted-public-api-changes.json

```diff
@@ -1,5 +1,21 @@
 {
     "acceptedApiChanges": [
+        {
+            "type": "org.gradle.api.tasks.SourceTask",
```

Comment:
Are we not concerned with these breaking changes? The prior APIs were not internal, right?

Reply:
The `Factory` interface is internal by itself. So if anyone was using these getters, they were using internal API.

While this is a breaking change as per binary-compatibility-check, I consider it very low risk in practice.

IMO, we should find a way for such changes to actually have no effect on the ABI. But then again we also want to make core tasks to not use any internal APIs eventually.

Reply:
Oh interesting, didn't even realize factory was internal